README.md

```diff
@@ -1,135 +1,170 @@
-# Diarizers
+# 🤗 Diarizers
 
-Diarizers is a library for fine-tuning [`pyannote`](https://github.com/pyannote/pyannote-audio/tree/develop) speaker diarisation models using Hugging Face tools. It provides scripts and tools to: 
-
-- Convert Speaker Diarization datasets into Hugging Face datasets compatible with the `diarizers` library. 
-- Fine-tune speaker diarization models using the Hugging Face `Trainer`. 
-- Test the models at inference time. 
-- Convert the fine-tuned models into a format compatible with `pyannote`.  
-
-> [!IMPORTANT]
-> For now, this library can only be used to fine-tune the [segmentation model](https://huggingface.co/pyannote/segmentation-3.0) from the [speaker diarization pipeline](https://huggingface.co/pyannote/speaker-diarization-3.1). 
-> Future work will aim to help optimise the hyperparameters of the entire pipeline. 
+🤗 Diarizers is a library for fine-tuning [`pyannote`](https://github.com/pyannote/pyannote-audio/tree/main) speaker 
+diarization models using the Hugging Face ecosystem. It can be used to improve performance on both English and multilingual 
+diarization datasets with simple example scripts, with as little as ten hours of labelled diarization data and just 5 minutes
+of GPU compute time.
 
 ## 📖 Quick Index
 * [Installation](#installation)
 * [Train](#train)
-* [Inference](#inference)
+* [Evaluation](#evaluation)
+* [Inference](#inference-with-pyannote)
 * [Results](#Results)
-* [Adding new datasets](#addingnewdatasets)
+* [Adding new datasets](#adding-new-datasets)
 * [Acknowledgements](#acknowledgements)
 * [Citation](#citation)
 
 ## Installation
 
-Diarizers has light dependencies and can be installed with the following lines: 
+First, clone the repository and install the dependencies:
```

Comment:
We need to clone the repository in order to get the example scripts. These won't be packaged as part of a pip install

Reply:
indeed!